src/Features/Core/Portable/Shared/IDocumentSupportsFeatureService.cs

```diff
@@ -39,5 +40,8 @@ public bool SupportsRefactorings(Document document)
 
         public bool SupportsRename(Document document)
             => true;
+
+        public bool SupportsSemanticSnippets(Document document)
+            => true;
```

Comment:
Does this need the `IsRazorDocument` check? I might be wrong, but I thought `ContainedDocument` was _only_ for legacy editor, but the new Razor LSP editor I would have guessed just uses this default service.